Mathlib/CategoryTheory/Enriched/EnrichedCat.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2025 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer
+-/
+import Mathlib.CategoryTheory.Enriched.Basic
+import Mathlib.CategoryTheory.Bicategory.Basic
+
+/-!
+# The bicategory of `V`-enriched categories
+
+We define the bicategory `EnrichedCat V` of (bundled) `V`-enriched categories for a fixed monoidal
+category `V`.
+
+## Future work
+
+* Define change of base and `ForgetEnrichment` as 2-functors.
+* Define the bicategory of enriched ordinary categories.
+-/
+
+
+universe w v u u₁ u₂
+
+namespace CategoryTheory
+
+open MonoidalCategory
+
+variable (V : Type v) [Category.{w} V] [MonoidalCategory V]
+
+/-- Category of `V`-enriched categories for a monoidal category `V`. -/
+def EnrichedCat := Bundled (EnrichedCategory.{w, v, u} V)
+
+namespace EnrichedCat
+
+instance : CoeSort (EnrichedCat V) (Type u) :=
+  ⟨Bundled.α⟩
+
+instance str (C : EnrichedCat.{w, v, u} V) : EnrichedCategory.{w, v, u} V C :=
+  Bundled.str C
+
+/-- Construct a bundled `EnrichedCat` from the underlying type and the typeclass. -/
+def of (C : Type u) [EnrichedCategory.{w} V C] : EnrichedCat.{w, v, u} V :=
+  Bundled.of C
+
+open EnrichedCategory ForgetEnrichment
+
+variable {V} {C D E E' : EnrichedCat.{w, v, u} V}
```

Comment:
Currently, definitions in this section eat bundled objects. I think it would be better to first make `C D E E'` regular types, equiped with an `EnrichedCategory` instance (with suitably general universes). This would give a general definition of the relevant whiskerings, associators and left/right unitors for general unbundled enriched functors, and only then it should be "bundled" as constructions in `EnrichedCat`. This would follow more closely the current design for functors of categories.
I missed this during first review, so apologies for bringing it up this late.